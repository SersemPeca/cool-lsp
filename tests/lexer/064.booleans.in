false
fAlse
faLse
falSe
falsE
fALse
faLSe
falSE
fAlSe
faLsE
fAlsE
fALSe
faLSE
fALsE
fAlSE
fALSE
