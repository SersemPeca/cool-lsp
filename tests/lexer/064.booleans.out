#1 BOOL_CONST false
#2 BOOL_CONST false
#3 BOOL_CONST false
#4 BOOL_CONST false
#5 BOOL_CONST false
#6 BOOL_CONST false
#7 BOOL_CONST false
#8 BOOL_CONST false
#9 BOOL_CONST false
#10 BOOL_CONST false
#11 BOOL_CONST false
#12 BOOL_CONST false
#13 BOOL_CONST false
#14 BOOL_CONST false
#15 BOOL_CONST false
#16 BOOL_CONST false
